src/app/admin-panel/admin-panel/correct-schedule/correct-schedule.component.scss

```diff
@@ -0,0 +1,84 @@
+//list of classes and download button
+.groups-list,
+.download-button {
+  margin: 1%;
+  margin-bottom: 2%;
+}
+
+.flex-container {
+  display: flex;
+  justify-content: center;
+}
+
+.flex-subjects-container {
+  display: flex;
+  justify-content: space-around;
+}
+
+.right-container {
+  margin-right: 1%;
+  width: 48%;
+}
+
+.left-container {
+  width: 48%;
+  margin-left: 1%;
+}
+
+:host ::ng-deep p-calendar {
+  margin-right: 3%;
+  margin-bottom: 2%;
+}
+
+h3 {
+  text-align: center;
+}
+
+// submit button
+p-footer .submit-subject-button {
+  width: 140px;
+  height: 38px;
+  margin-top: 2%;
+  margin-left: 1.5%;
+  margin-bottom: 5%;
+}
+
+:host ::ng-deep p-footer .submit-subject-button .ui-button-icon-left {
+  margin-left: 17px;
+}
+
+@media screen and (max-width: 1260px) {
+  :host ::ng-deep .ui-datepicker {
+    display: block;
+    margin-left: -18%;
```

Comment:
are you sure that negative margin will be ok on every screen?

Reply:
I've changed styles